objects/src/accounts/code.rs

```diff
@@ -158,7 +158,8 @@ impl Eq for AccountCode {}
 impl Serializable for AccountCode {
     fn write_into<W: ByteWriter>(&self, target: &mut W) {
         self.module.write_into(target, MODULE_SERDE_OPTIONS);
-        self.module.write_source_locations(target);
+        // FIXME: this is failing even when serializing and de-serializing a proven mint tx
+        // self.module.write_source_locations(target);
```

Comment:
I'm not sure this should be a `FIXME` - I think by default we don't need to serialize source locations since this is basically debug info. For that matter, we probably don't need to serialize imports either - so, maybe `MODULE_SERDE_OPTIONS` should be changed. I'd probably just leave a comment here to explicitly state that we do not serialize imports and source locations with account code.

Reply:
changed it for a comment mentioning that. I also changed `MODULE_SERDE_OPTIONS` to have a `serialize_imports: false` by default. I'm not sure you meant to change it on this PR though